docs/prosessere-data/notebook_general.md

```diff
@@ -67,9 +67,10 @@ Du kan koble deg til VMen som notebook serveren din kjører på med SSH fra VS C
 
 1. Først trenger du å få `owner` rettighet på VM instansen. Dette får du ved å ta kontakt i [#nada](https://nav-it.slack.com/archives/CGRMQHT50).
 2. Installer extension `Remote - SSH` i VS Code.
-3. Hvis du ikke har gjort det i dag, kjør kommandoen `gcloud auth login --update-adc`.
-4. Kjør kommandoen `gcloud compute ssh --project knada-gcp --zone europe-west1-b <instance> --dry-run`. Erstatt `<instance>` med navnet på VM instansen din, denne finner du [her](https://console.cloud.google.com/compute/instances?project=knada-gcp). Denne kommandoen vil også generere ssh nøkler.
-5. Outputen fra kommandoen i (4) inneholder en del ting du trenger fylle inn i ssh-configen din. Under er et eksempel på hvordan en slik ssh config skal se ut.
+3. Hvis du ikke har gjort det i dag, kjør kommandoen `gcloud auth login --update-adc`. Hvis dette ikke fungerer last ned gcloud CLI https://cloud.google.com/sdk/docs/install-sdk og prøv igjen.
+4. 5. Kjør kommandoen `gcloud compute ssh --project knada-gcp --zone europe-west1-b <instance> Erstatt `<instance>` med navnet på VM instansen din, denne finner du [her](https://console.cloud.google.com/compute/instances?project=knada-gcp) under "Name".
```

Comment:
Hvorfor skal man kjøre denne to ganger?

Reply:
Jeg måtte kjøre den uten --dry-run først og så etterpå med --dry-run.